server/models/organizer.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const organizerSchema = new Schema({
+  name: {
+    type: Schema.Types.ObjectId,
```

Comment:
I see this goes to the "name" schema, is that what we want for organization, despite that name.js seems to be build for user name